### PR TITLE
First wave of additions

### PR DESCRIPTION
**Pull request by kelnera**
This is the first set of updates to the CASS student page

### CODE REVIEW

**Comment by kelnera** [comment]
@Kennric @osu-cass/cass-media @ShannaChang @morgaalb @bitschift @audrirock @Zainkai @vandomej @danvanhorn If you've been tagged, you're either on the media team or your name got added to the student page. If I could get some plus ones for this, that would be great.

File: content/pages/organization/cass-students.rst [hunk 1]
```diff
@@ -534,6 +664,38 @@ https://github.com/tschuy
 
   <br/>
 
+Kevin Turkington
+----------------
+
+.. image:: kturkington.jpg
```
Comment (ramereth) [inline content/pages/organization/cass-students.rst]: I believe the image URL for this is broken.

File: content/pages/organization/cass-students.rst [hunk 2]
```diff
@@ -273,7 +323,77 @@ drums in his free time.
...
+real-world application. In his spare time, Aravind likes to read and play video
+games, such as Deus Ex: Human Revolution. He is also a polyglot!
+
+.. class:: no-breaks
+
+  **Github Account**
+
+https://github.com/bitschift
+
+..class:: no-breaks
```
Comment (ramereth) [inline content/pages/organization/cass-students.rst]: Missing a space in front of ``class``.

File: content/pages/organization/cass-students.rst [hunk 3]
```diff
@@ -584,6 +746,53 @@ https://github.com/mathuin
...
+.. class:: no-breaks
+
+  **Software Developer/Analyst**
+
+Dan came to CASS in June of 2016 and is majoring in Computer Science. Dan is
+passionate about learning and constantly expanding his skills in software
+development and he applied to CASS as a way to feed this passion. His favorite
+projects are ones that challenge him to find efficient solutions to difficult
+problems. In his spare time, Dan likes to rock climb and cycle.
+
```
Comment (ramereth) [inline content/pages/organization/cass-students.rst]: Missing ``.. class:: no-breaks``

**Comment by hertelc** [comment]
I can't help with the code (yet!), but I did read the paragraphs for typos and they look good to me.

File: content/pages/organization/cass-students.rst (hunk 1 above)
Comment (kelnera) [inline content/pages/organization/cass-students.rst]: I just forgot some of the code. It has to say /images/picturename. Hopefully it works now.

**Comment by ramereth** [comment]
@hertelc great work! that fixed the issues I think.

:+1:

**Comment by hertelc** [comment]
@Kennric  Is this ready to merge and go live?  Lets get it pushed please.

**Pull request merged**